tools/code-owners-parser/Azure.Sdk.Tools.RetrieveCodeOwners.Tests/CodeownersManualAnalysisTests.cs

```diff
@@ -352,6 +352,7 @@ private void WriteLangRepoOwnersDiffToCsv(string langName)
         outputLines.AddRange(PathsWithMissingPrefixSlash(entries));
         outputLines.AddRange(PathsWithMissingSuffixSlash(targetDir, entries, paths));
         outputLines.AddRange(InvalidPaths(entries));
+        // TODO: add a check here for CODEOWNERS paths that do not match any dir or file.
```

Comment:
Do we need an issue tracking this?

Reply:
@benbp not really. I put it here more as a reminder that this validation is currently missing, in the unlikely scenario I (or somebody else) will have to use it again. I forgot at first to include it. But I kind of expect for it to be actually picked up when working on:
- https://github.com/Azure/azure-sdk-tools/issues/4859